release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2478,6 +2478,145 @@ index a90ab14ff..f7f6f1a71 100644
+
+. Pull the `rhcos-live` ISO image for the release being installed. The URL can be retrieved from the `AgentServiceConfig` CR on the hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get AgentServiceConfig agent -o yaml
+----
+
+. Mount the image in the `/mnt/iso/` directory:
++
+[source,terminal]
+----
+$ mount rhcos-live.x86_64.iso /mnt/iso/
+----
+
+. Create the `iso-grub-cfg/` directory and change to the directory:
++
+[source,terminal]
+----
+$ mkdir iso-grub-cfg/; pushd iso-grub-cfg/
+----
+
+. Copy the contents of the `/mnt/iso/` directory to your working directory:
++
+[source,terminal]
+----
+$ rsync -avH /mnt/iso/* .
+----
+
+. Open the GRUB configuration file:
++
+[source,terminal]
+----
+$ vim EFI/redhat/grub.cfg
+----
+
+.. Append the `rd.net.timeout.carrier=20` string to the `linux` boot line.
+
+. Return to the initial working directory:
++
+[source,terminal]
+----
+$ popd
+----
+
+. Generate the ISO file from the `iso-grub-cfg` directory:
++
+[source,terminal]
+----
+$ mkisofs -JR -graft-points -o rhcos-carrier-timeout.iso iso-grub-cfg
+----
+
+. Push the updated ISO image to the server that is accessible by the hub cluster.
+
+. On the hub cluster, update the `osImages` entry in the `AgentServiceConfig` CR for the installing release to point to the updated ISO image:
++
+[source,terminal]
+----
+$ oc edit AgentServiceConfig agent
+----
+
+. Update the `url` field to point to the URL of the updated ISO image.
+
++
+(link:https://issues.redhat.com/browse/OCPBUGSM-46336[*OCPBUGSM-46336*])
+
+* A kernel crash occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+spec:
+  additionalKernelArgs:
+  - rcutree.kthread_prio=11
+----
++
+(link:https://issues.redhat.com/browse/RHELPLAN-123262[*RHELPLAN-123262*])
+
+* During ZTP cluster deployments, the bare-metal host image provisioning fails with the following error: `The ETag supplied did not match the ETag required to change this resource`. To workaround this issue, attach the ISO image to the host using the Baseboard Management Controller GUI. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])
+
+* During a master node replacement in a bare-metal cluster, the new master host gets stuck in a `Provisioning` state, but the node reports in to the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])
+
+* At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` datastore service uses isolated application cores. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])
```

Comment:
```suggestion
* At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` data store service uses isolated application cores. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])
```

Reply:
See the ISG entry for 'datastore': https://www.ibm.com/docs/en/ibm-style?topic=word-usage